testing/ui-tests/src/incorrect/substitute_path_not_absolute.stderr

```diff
@@ -1,4 +1,4 @@
-error: The substitute path must be a global absolute path; try prefixing with `::` or `crate`
+error: Type substitution error: The 'to' type must be a path prefixed with 'crate::' or '::'
```

Comment:
This error could be better, `to` is not super friendly.

I would prefer:  "the replaced type must have a path prefix with `crate::` or `::`" but